### PR TITLE
Mui integration(fix validation)

### PR DESCRIPTION
**Pull request by Thundem**
Link to video:
https://www.awesomescreenshot.com/video/27355980?key=79cc78b8079b7101ea63e69e76462065

### CODE REVIEW

**Comment by Emil-Khalus** [comment]
In your video, you can see that a validation error is displayed after the summit. What do you think about clearing these inputs after the form submission and validating them only when the user edits them?
image

**Review by Emil-Khalus — CHANGES_REQUESTED** [review]

**Comment by Thundem** [comment]
I already fixed it

File: src/components/styledComponents/StyledComponent.tsx
[the remark this answers is not in the document]
Comment (Thundem) [reply src/components/styledComponents/StyledComponent.tsx:1]: If I understand correctly, i need to move each styled components in separate files?

**Comment by Thundem** [comment]
I already fixed bug with startRentTime component, but I forgot refresh my video

**Review by mykyta07 — APPROVED** [review]

**Pull request merged**